website/docs/r/monitor_alert_prometheus_rule_group.html.markdown

```diff
@@ -157,7 +154,7 @@ An `action` block supports the following:
 
 * `action_group_id` - (Required) Specifies the resource id of the monitor action group.
 
-* `action_properties` - (Optional) Specifies the properties of an action group object.
+* `action_properties` - (Optional) Specifies the properties of an action group object. Only available for IcM Connector action group.
```

Comment:
```suggestion
* `action_properties` - (Optional) Specifies the properties of an action group object.

-> **Note:** `action_properties` can only be configured for IcM Connector Action Groups.
```

Reply:
@teowa so should this field be supported at all on this resource?

Reply:
From service team, the `action_properties` might expose new public properties in the future, so we still need to support this.